scss/_ps-dropdown.scss

```diff
@@ -0,0 +1,64 @@
+.ps-dropdown {
+    font-size: 1em;
```

Comment:
Watch out with ems, this could cause trouble

Reply:
I'll take a look into it but it's a new dropdown from the core itself, didn't touch everything

Reply:
best use `rem` so that it isn't affected by font size from the parent node.